internal/web_package/injector.js

```diff
@@ -39,6 +39,18 @@ function findElementByName(d, name) {
 function main(params, read = fs.readFileSync, write = fs.writeFileSync, timestamp = Date.now) {
   const outputFile = params.shift();
   const inputFile = params.shift();
+  const rootDirs = [];
+  while (params.length && params[0] !== '--assets') {
+    let r = params.shift();
+    if (!r.endsWith('/')) {
```

Comment:
Maybe we can use `path.sep` which returns the correct separator depending on the OS.

Reply:
these paths are constructed by bazel so they should be posix paths always